cmd/minishift/cmd/root.go

```diff
@@ -68,7 +72,15 @@ var RootCmd = &cobra.Command{
 	Short: "Minishift is a tool for application development in local OpenShift clusters.",
 	Long:  `Minishift is a command-line tool that provisions and manages single-node OpenShift clusters optimized for development workflows.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		var err error
+		var (
+			err          error
+			isFreshStart bool
+		)
+
+		if filehelper.IsEmptyDir(constants.Minipath) {
```

Comment:
So in the light of https://github.com/minishift/minishift/pull/990/files#r121346230, this might need to be 

    if ! filehelper.Exists(constants.Minipath) || filehelper.IsEmptyDir(constants.Minipath)